gatherers/manifest.js

```diff
@@ -71,7 +72,11 @@ class Manifest extends Gather {
           })
           .then(manifestURL => Manifest._loadFromURL(options, manifestURL));
         })
-        .then(manifest => ({manifest}));
+        .then(manifestContent => {
+          return {
+            manifest: manifestParser(manifestContent).value
```

Comment:
In the current version both equate to a failed audit. I think we should move to "nuanced" fairly soon, especially as the perf numbers start to come in.